_observing-your-data/notifications/index.md

```diff
@@ -100,6 +100,9 @@ Then add this policy into the IAM user’s trust relationship to actually assume
 }
 ```
 
+### Host deny list
+
+To specify any IP address ranges or hostnames for which the nodes running the OpenSearch software should not make requests against, populate the host deny list:
```

Comment:
@toepkerd A list of information appears to be missing after the colon `:`. Also, we should include a link to the notifications developer guide https://github.com/opensearch-project/notifications/blob/main/DEVELOPER_GUIDE.md. Is there info about the deny list in there too?

Reply:
Is the deny list purpose to mute or reduce alerts?